### DISCUSSION TITLE
Github Student Developer Pack

### DISCUSSION THREAD

**Discussion by OmarKhalidAldehoon** [community/community#61492] [GitHub Education]
### Hi everyone,

Hi, I recently applied for a request to get the benefits of the student developer pack, also my request has been approved but the benefits are not available yet!!
![Request-a-discount-GitHub-Education](https://github.com/community/community/assets/133169152/d3f14f87-ec54-40ce-b8cd-cab3831c491b)

**Comment by es1228** [comment]
Same here, applied July 20. Some people say it takes up to 5 days so if it takes longer than there might be a problem.

**Reply by OmarKhalidAldehoon** [reply]
They said it would take at most 4 days when I applied.

**Reply by es1228** [reply]
I got it, seems like it takes 3 days or more
